test/story-nft/StoryBadgeNFT.t.sol

```diff
@@ -116,7 +116,7 @@ contract StoryBadgeNFTTest is BaseTest {
         );
     }
 
-    function test_StoryBadgeNFT_interfaceSupport() public {
+    function test_StoryBadgeNFT_interfaceSupport() public view {
```

Comment:
the `view` is not needed.

Reply:
resolved in [6fd5f9a](https://github.com/storyprotocol/protocol-periphery-v1/pull/89/commits/6fd5f9a1138f7775dc041ad05d6c757473acb3d4)